kolibri/plugins/management/assets/src/vue/user-page/user-edit-modal.vue

```diff
@@ -2,22 +2,47 @@
 
   <div class="user-edit-modal">
     <modal btntext="Edit">
-      <div class="title" slot="header">User edit</div>
+
+      <h1 slot="header" class="header">Account Info</h1>
+
       <div slot="body">
-        <br>Username: <input type="text" v-model="username_new" placeholder="Please type in your username."><br>
-        <br>Password: <input type="text" v-model="password_new" placeholder="Please type in your password."><br>
-        <br>Full name: <input type="text" v-model="fullName_new" placeholder="Please type in your first name."><br>
-        <!-- radio buttons for selecting role -->
-        <br>Learner <input type="radio" value="learner" v-model="role_new"><br>
-        <br>Admin <input type="radio" value="admin" v-model="role_new"><br>
-      </div>
-      <div slot="footer">
-        <button class="confirm-btn" type="button" @click="editUser">Confirm</button>
-      </div>
-      <div slot="openbtn">
-        <svg class="manage-edit" src="../icons/pencil.svg"></svg>
+
+        <div class="user-field">
+          <label for="username">Full Name</label>:
+          <input type="text" class="edit-form edit-fullname" aria-label="fullname" id="name" v-model="fullName_new">
+        </div>
+
+        <div class="user-field">
+          <label for="name">Username</label>:
+          <input type="text" class="edit-form edit-username" aria-label="username" id="username" v-model="username_new">
+        </div>
+
+        <div class="user-field">
+          <label for="user-role"><span class="visuallyhidden">User Role</span></label>
+          <select v-model="user.role" id="user-role">
+          <option value="learner" selected> Learner </option>
+          <option value="admin"> Admin </option>
+          </select>
+        </div>
+
+        <div class="user-field">
+        <label for="password">Password</label>:
+        <input type="text" id="password" v-model="password_new" placeholder="Please type in your password.">
+        </div>
+
       </div>
+
+        <div slot="footer">
+          <button class="confirm-btn" type="button" @click="editUser">Confirm</button>
+        </div>
+
+        <button class="no-border" slot="openbtn">
+          <span class="visuallyhidden">Edit User</span>
```

Comment:
Could you change the wording here to "Edit Account Info" too, since this is the button that opens the modal? :smiley: